### PR TITLE
[new] qStudio integrations

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

Tested and validated the qStudio integrations.
# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

**Comment by Rajakavitha1** [comment]
Hi @jdjohn555  and @Loquacity !!!nI was not able to commit my changes in to the branch for https://github.com/timescale/docs/pull/2585 because it was forked. I have tested and Validated the instructions and they work for Timescale Cloud service and Self-hosted instance.

**Comment by jdjohn555** [comment]
Nice work @Rajakavitha1  - I wasn't sure how to add images.

File: use-timescale/integrations/query-admin/qstudio.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+keywords: [connect]
+---
+
+# Connect to Timescale using qStudio
+
+[qStudio][qstudio] is a free SQL GUI to run SQL scripts, browse the tables,
+create charts and export the results. You can install qStudio on every operating
+system and connect to every database including Timescale. You can connect to
+Timescale hosted on your local machine or on a remote server or a Timescale
+service.
```
Comment (Loquacity) [inline use-timescale/integrations/query-admin/qstudio.md:14]:
```suggestion
You can use [qStudio][qstudio] to run queries, browse tables, and create charts for your Timescale database.
```

File: use-timescale/integrations/query-admin/qstudio.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+<Tab title="Timescale">
+
+<Procedure>
+
+1.  Sign in to the [Timescale portal][tsc-portal].
+1.  In the `Services` tab, find the service you want to connect to, and check
+    it is marked as `Running`.
+1.  Click the name of the service you want to connect to, and take a note
+     of the `Host`, `Port`, `Database name`, and `Username`.
```
Comment (Loquacity) [inline use-timescale/integrations/query-admin/qstudio.md:33]:
```suggestion
1.  Sign in to the [Timescale portal][tsc-portal], and navigate to the `Services` 
     tab. For the service you want to connect to, check it is marked as `Running`, 
     and take a note of the `Host`, `Port`, `Database name`, and `Username`.
```

File: use-timescale/integrations/query-admin/qstudio.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    to the `Operations` tab, and click `Reset password`. You can choose your own
+    password for the service, or allow Timescale to generate a
+    secure password for you. Take a note of your new password.
+1.  Start `qStudio`.
+1.  In the `Server` tab click `Add Server`.
+1.  In the `Server Properties` dialog select `Server Type` as `Postgres`.
+1.  Set `Connect By` as `URL`
+1.  Set the `Host`, `Port`,`Database`, `Username` and `Password`.For more
+    information about these settings,
+    see [the connection section][connection-details].
```
Comment (Loquacity) [inline use-timescale/integrations/query-admin/qstudio.md:44]:
```suggestion
1.  Start `qStudio`, navigate to the `Server` tab, and click `Add Server`.
1.  In the `Server Properties` dialog, complete these details:
    *   For `Server Type`, select `Postgres`.
    *   For `URL`, select `Connect By`.
    *   For the `Host`, `Port`,`Database`, `Username` and `Password`, use 
         your Service connection details. For more information about these 
         settings, see [the connection section][connection-details].
```

File: use-timescale/integrations/query-admin/qstudio.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+1.  Click `Test` to check if the connection is successful.
+1.  Click `Add` to add the database server.
```
Comment (Loquacity) [inline use-timescale/integrations/query-admin/qstudio.md:46]:
```suggestion
1.  Click `Add` to add the connection.
```

File: use-timescale/integrations/query-admin/qstudio.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+1.  Start `qStudio`.
+1.  In the `Server` tab click `Add Server`.
+1.  In the `Server Properties` dialog select `Server Type` as `Postgres`.
+1.  Set `Connect By`as `Host`.
+1.  Set the `Host`, `Port`,`Database`, `Username` and `Password`. For more
+    information about these settings,
+    see [the connection section][connection-details].
+1.  Click `Test` to check if the connection is successful.
+1.  Click `Add` to add the database server.
+    The server is listed in the `Server Tree` pane.
```
Comment (Loquacity) [inline use-timescale/integrations/query-admin/qstudio.md:70]: Use the bullets for completing the dialog, like the other procedure. ☝️

File: use-timescale/integrations/query-admin/qstudio.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+    src="https://s3.amazonaws.com/assets.timescale.com/docs/images/qstudio_self_hosted.webp"
+    alt="Connecting to Timescale using qstudio"/>
+
+</Procedure>
+
+</Tab>
+
+</Tabs>
+
+## Settings in `Connection`
```
Comment (Loquacity) [inline use-timescale/integrations/query-admin/qstudio.md:82]:
```suggestion
## Connection settings
```

File: use-timescale/integrations/query-admin/qstudio.md (hunk 5 above)
Comment (Rajakavitha1) [reply use-timescale/integrations/query-admin/qstudio.md]: Done!!! Makes sense :)

File: use-timescale/integrations/query-admin/qstudio.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+1.  [](#)<Optional />If you do not know the password for the service, navigate
+    to the `Operations` tab, and click `Reset password`. You can choose your own
+    password for the service, or allow Timescale to generate a
+    secure password for you. Take note of your new password.
+1.  Start `qStudio`, navigate to the `Server` tab, and click `Add Server`.
+1.  In the `Server Properties` dialog, complete these details:
+    *   For `Server Type`, select `Postgres`.
+    *   For `Connect By`, select `Host`.
+    *   For the `Host`, `Port`, `Database`, `Username` and `Password`, use 
+         your Service connection details. For more information about these
```
Comment (Loquacity) [inline use-timescale/integrations/query-admin/qstudio.md:38]:
```suggestion
         your service connection details. For more information about these 
```

File: use-timescale/integrations/query-admin/qstudio.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+
+<Procedure>
+
+1.  Start `qStudio`.
+1.  In the `Server` tab click `Add Server`.
+1.  In the `Server Properties` dialog, complete these details:
+    *   For `Server Type`, select `Postgres`.
+    *   For `Connect By`, select `Host`.
+    *   For the `Host`, `Port`, `Database`, `Username` and `Password`, use 
+         your Service connection details. For more information about these
```
Comment (Loquacity) [inline use-timescale/integrations/query-admin/qstudio.md:62]:
```suggestion
         your service connection details. For more information about these 
```

File: use-timescale/integrations/query-admin/qstudio.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+</Tabs>
+
+## Connection settings
+
+This table provides the description and example values for the fields in
+`Connection` dialog of `Server Properties`.
+
+|Setting|Example value|Description|
+|-|-|-|
+|Host|`localhost`, `postgresql.example.com`, `<REMOTE_HOST>.tsdb.cloud.timescale.com`|The fully qualified server name or IP address of your TimescaleDB server or the host name of your Timescale service.|
```
Comment (Loquacity) [inline use-timescale/integrations/query-admin/qstudio.md:85]:
```suggestion
|Host|`localhost`, `postgresql.example.com`, `<REMOTE_HOST>.tsdb.cloud.timescale.com`|The fully qualified server name or IP address of your TimescaleDB server, or the hostname of your Timescale service.|
```

File: use-timescale/integrations/query-admin/qstudio.md [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+|Port|`5432`|The port number of the TimescaleDB server, Timescale service.|
```
Comment (Loquacity) [inline use-timescale/integrations/query-admin/qstudio.md:86]:
```suggestion
|Port|`5432`|The port number of the TimescaleDB server, or Timescale service.|
```

File: use-timescale/integrations/query-admin/qstudio.md [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+|Username|`postgres`, `tsdbadmin`|The user name you want to log in with. Use `tsdbadmin` for Timescale services|
```
Comment (Loquacity) [inline use-timescale/integrations/query-admin/qstudio.md:87]:
```suggestion
|Username|`postgres`, `tsdbadmin`|The user name you want to log in with. Use `tsdbadmin` for Timescale services.|
```

File: use-timescale/integrations/query-admin/qstudio.md [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+|Password|*password*|The password for the database user you are connecting to.|
```
Comment (Loquacity) [inline use-timescale/integrations/query-admin/qstudio.md:88]:
```suggestion
|Password||The password for the database user you are connecting to.|
```

File: use-timescale/integrations/query-admin/qstudio.md [hunk 13]
```diff
@@ -0,0 +1,94 @@
...
+|Database|`tsdb`|This option connects to the default database. The database name for a Timescale service is `tsdb`. You can also specify the name of the database to connect to.|
```
Comment (Loquacity) [inline use-timescale/integrations/query-admin/qstudio.md:89]:
```suggestion
|Database|`tsdb`|The name of the database to connect to. For a Timescale service, use `tsdb`.|
```

Comment (Loquacity) [reply use-timescale/integrations/query-admin/qstudio.md:89]: I suspect we can get some more parallelism into this table with some effort 🤔

**Review by Loquacity — COMMENTED** [review]
Sorry, clearly missed the table on the first pass.

**Comment by Rajakavitha1** [comment]
> Sorry, clearly missed the table on the first pass.

no worries :)

**Review by Loquacity — APPROVED** [review]

**Pull request merged**